valuesets/ValueSet-England-BusinessChangeDateTimeType.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+	<id value="England-BusinessChangeDateTimeType"/>
+	<url value="https://fhir.nhs.uk/England/ValueSet/England-BusinessChangeDateTimeType"/>
+	<version value="1.1.0"/>
+	<name value="EnglandBusinessChangeDateTimeType"/>
+	<title value="England Business Change Date Time Type"/>
+	<status value="draft"/>
+	<date value="2023-10-23"/>
+	<publisher value="NHS England"/>
+	<contact>
+		<name value="NHS England"/>
+		<telecom>
+			<system value="email"/>
+			<value value="interoperabilityteam@nhs.net"/>
+			<use value="work"/>
+			<rank value="1"/>
+		</telecom>
+	</contact>
+	<description value="A ValueSet to identify the type of business change date and time being recorded."/>
+	<copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html."/>
+	<compose>
+		<include>
+			<system value="https://fhir.nhs.uk/England/CodeSystem/England-BusinessChangeDateTimeType"/>
+		</include>
+	</compose>
+	<expansion>
+		<identifier value="0782b2cc-5813-434e-bb6c-1cf79f2ca01b"/>
```

Comment:
Does this identifier need to be here?

Reply:
Not essential, but not a blocker, it's present on UK Core expansions